src/main/java/doit/jpastudy2/repository/CategoryRepository.java

```diff
@@ -10,8 +10,8 @@ public interface CategoryRepository extends JpaRepository<Category, Long> {
     // find + [ ] + By + (조건)
 
     // select * from Category
-    Category findByDescription(String description);
+    // Category findByDescription(String description);
 
     // select * from Category where type = ? and description = ?
-    Category findByTypeAndDescription(String type, String description);
+    // Category findByTypeAndDescription(String type, String description);
 }
```

Comment:
우선 클래스까지는 잘 생성해주셨습니다.
다만 이 쪽에서 JpaRepository를 사용하기 위해서는, JpaRepository<Entity Class 이름, 그 클래스의 PK 타입>으로 작성해야합니다. 현재까지 작성해주신 Entity는 String 타입이어서 아래와 같이 작성할 수 있습니다.
```suggestion
public interface CategoryRepository extends JpaRepository<Category, String> {
```